src/normalizer/mod.rs

```diff
@@ -57,13 +65,35 @@ impl<'o> Iterator for NormalizedTokenIter<'o> {
     }
 }
 
+/// Structure for providing options to a normalizer.
+#[derive(Clone)]
+pub struct NormalizerOption {
+    pub create_char_map: bool,
+}
+
+impl NormalizerOption {
+    
+    /// Creates a new [`NormalizerOption`] with default options set.
+    pub fn new() -> NormalizerOption {
+        NormalizerOption {
+            create_char_map: true,
+        }
+    }
+}
```

Comment:
You should [Implement the `Default` trait](https://doc.rust-lang.org/std/default/trait.Default.html) instead.